subschemas/substrate-chain/src/resolvers/Query/auctions.ts

```diff
@@ -1,8 +1,7 @@
-
 import {ApiPromise} from '@polkadot/api';
 import {createWsEndpoints} from '@polkadot/apps-config/endpoints';
 import type {LinkOption} from '@polkadot/apps-config/endpoints/types';
-import type {ITuple, Registry, Codec} from '@polkadot/types/types';
```

Comment:
This was just an unused leftover. Other changes in this file are only prettier formatting :)